src/bin/sol-fbp-runner/web-inspector.js

```diff
@@ -16,451 +16,178 @@
+        function feed(data){
+            ev = JSON.parse(data);
+            this.fbpInspector.feedEvent(ev);
         }
-    }
-
-    return '<span class="fbp-packet fbp-packet-type-' + packet_type + ' ' + extraClass + '">' + body + '</span>';
+    };
 };
 
-FBPInspector.prototype.renderOptionValue = function (data_type, value) {
-    var extraClass = "";
-    var body = "";
-
-    if (data_type == "boolean" ||
-               data_type == "byte" ||
-               data_type == "timestamp") {
-        body = value;
-    } else if (data_type == "string") {
-        body = value; // TODO: escape
-    } else {
-        var keys = [];
-        for (var key in value) {
-            if (!value.hasOwnProperty(key))
-                continue;
-            keys.push(key);
-        }
-
-        keys.sort();
-
-        for (var i = 0; i < keys.length; i++) {
-            var key = keys[i];
-            var val = value[key];
-            if (i > 0)
-                body += '<span class="fbp-node-option-field-separator">, </span>';
-            body += '<span class="fbp-node-option-field">' +
-                '<span class="fbp-node-option-field-key">' + key + ': </span>' + // TODO: escape
-                '<span class="fbp-node-option-field-value">' + val + '</span>' + // TODO: escape
-                '</span>';
-        }
-
-        if (data_type == "rgb" && value) {
-            var r = Math.round(255 * (value.red / value.red_max));
-            var g = Math.round(255 * (value.green / value.green_max));
-            var b = Math.round(255 * (value.blue / value.blue_max));
-
-            body += '<span class="fbp-node-option-type-rgb-preview" style="background-color: rgb(' + r + ',' + g + ',' + b + ');"></span>';
-        }
+FBPInspector.prototype.finish = function () {
+    if (this.source) {
+        this.source.close();
+        this.source = null;
     }
+};
 
-    return '<span class="fbp-node-option fbp-node-option-type-' + data_type + ' ' + extraClass + '">' + body + '</span>';
-}
-
-FBPInspector.prototype.renderNodeInfo = function (node) {
-    var n = this.nodes[node];
-    var payload = n.creation.payload;
-    var creationTimestamp = n.creation.timestamp;
-    var destructionTimestamp = n.destruction;
+FBPInspector.prototype.logSentClear = function () {};
 
-    // TODO: hadle escaping of html
+FBPInspector.prototype.logDeliveredClear = function () {};
 
-    var html = '<h3>' + this.getNodeName(node) + '</h3>' +
-        '<dl id="fbp-node-info-' + node + '">';
+FBPInspector.prototype.feedEvent = function (data) {
 
-    if (payload.type) {
-        html += '<dt class="fbp-node-type">type</dt>' +
-            '<dd class="fbp-node-type">' + payload.type + '</dd>';
-    }
+    if (data.event == "open") {
 
-    if (payload.category) {
-        html += '<dt class="fbp-node-category">category</dt>' +
-            '<dd class="fbp-node-category">' + payload.category + '</dd>';
-    }
+        if(data.payload.id.indexOf(".fbp") > -1) return;
```

Comment:
this is because it's the root node, I'd say do not use `data.payload.id` but `data.payload.path.length == 1`.